apps/pdc-frontend/src/styles/print.css

```diff
@@ -165,6 +165,12 @@
       display: none;
     }
 
+    /* Footer color */
+    footer,
+    .utrecht-footer * {
```

Comment:
De Utrecht Page Footer heeft al code om diverse dingen in te stellen op de huidige kleur:

https://github.com/nl-design-system/utrecht/blob/main/components/page-footer/src/_mixin.scss#L14-L21

Wat werkt er nog niet wanneer je `--utrecht-page-footer-color: var(--utrecht-color-black);` doet? Waaorm is deze extra code nodig?